extensions/confluence-search/src/index.tsx

```diff
@@ -0,0 +1,205 @@
+export const confluenceUrl = `https://${prefs.instance}`;
+
+const headers = {
+  Accept: "application/json",
+  Authorization: "Basic " + Buffer.from(`${prefs.user}:${prefs.token}`).toString("base64"),
+};
+
+const init = {
+  headers,
+};
+
+export default function Command() {
+  const [results, setResults] = useState<SearchResult[]>([]);
+
+  const [loadingState, setLoadingState] = useState(true);
+
+  useEffect(() => {
+    searchConfluence().then((response) => {
+      setLoadingState(false);
+      if (!response.ok) {
+        const failureMessage = response.message ? response.message : response.statusText;
+        showToast(ToastStyle.Failure, "API request failed", failureMessage);
+      } else {
+        parseResponse(response).then((response: SearchResult[]) => {
+          setResults(response);
+        });
+      }
+    });
+  }, []);
+
+  return (
+    <List isLoading={loadingState} searchBarPlaceholder="Search by name..." throttle>
+      <List.Section title="Results">
+        {results && results.map((searchResult) => <SearchListItem key={searchResult.id} searchResult={searchResult} />)}
+      </List.Section>
+    </List>
+  );
+}
+
+async function searchConfluence() {
+  const apiUrl = `${confluenceUrl}/wiki/rest/api/content?expand=version`;
+  const response = await fetch(apiUrl, init)
+    .then((response) => {
+      return response;
+    })
+    .catch((error) => {
+      {
+        return error;
+      }
+    });
+
+  return response;
+}
+
+async function parseResponse(response: Response) {
+  const json = (await response.json()) as APIResponse;
+  const jsonResults = (json?.results as ResultsItem[]) ?? [];
+  return jsonResults.map((jsonResult: ResultsItem) => {
+    return {
+      id: jsonResult.id as string,
+      name: jsonResult.title as string,
+      type: jsonResult.type as string,
+      url: jsonResult._links.webui as string,
+      author: jsonResult.version.by.displayName as string,
+      icon: jsonResult.version.by.profilePicture.path as string,
+    };
+  });
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.name}
+      subtitle={searchResult.type}
+      keywords={[searchResult.name, searchResult.type]}
+      accessoryTitle={searchResult.author}
+      accessoryIcon={{ source: `${confluenceUrl}${searchResult.icon}`, mask: ImageMask.Circle }}
+      icon={Icon.Document}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <OpenInBrowserAction title="Open in Browser" url={confluenceUrl + "/wiki" + searchResult.url} />
+          </ActionPanel.Section>
+          <ActionPanel.Section>
+            <CopyToClipboardAction
+              title="Copy URL"
+              content={confluenceUrl + "/wiki" + searchResult.url}
+              shortcut={{ modifiers: ["cmd"], key: "." }}
+            />
+          </ActionPanel.Section>
+        </ActionPanel>
```

Comment:
```suggestion
        <ActionPanel>
          <ActionPanel.Section>
            <OpenInBrowserAction title="Open in Browser" url={confluenceUrl + "/wiki" + searchResult.url} />
            <CopyToClipboardAction
              title="Copy URL"
              content={confluenceUrl + "/wiki" + searchResult.url}
              shortcut={{ modifiers: ["cmd"], key: "." }}
            />
          </ActionPanel.Section>
        </ActionPanel>
```
You could put it in the same section